### PR TITLE
docs: Added recommendation badges

### PR DESCRIPTION
**Pull request by somehybrid**
Added recommendation badges as per #272.

### CODE REVIEW

**Comment by newpavlov** [comment]
The badge takes too much space in my opinion. I think it's better to use something like the [hashes security column](https://github.com/RustCrypto/hashes?tab=readme-ov-file#supported-algorithms). Also, I don't think we not use githubusercontent links in README.

**Comment by somehybrid** [comment]
> The badge takes too much space in my opinion. I think it's better to use something like the [hashes security column](https://github.com/RustCrypto/hashes?tab=readme-ov-file#supported-algorithms). Also, I don't think we not use githubusercontent links in README.

Hi, sorry for the long response. I changed it into a similar style. Is this okay?

File: README.md [hunk 1]
```diff
@@ -16,17 +16,25 @@ crate.
...
+| [`eax`]              | [EAX]                 | [![crates.io](https://img.shields.io/crates/v/eax.svg)](https://crates.io/crates/eax)                           | [![Documentation](https://docs.rs/eax/badge.svg)](https://docs.rs/eax)                           | 1.41 | 💚       |
+| [`mgm`]              | [MGM]                 | [![crates.io](https://img.shields.io/crates/v/mgm.svg)](https://crates.io/crates/mgm)                           | [![Documentation](https://docs.rs/mgm/badge.svg)](https://docs.rs/mgm)                           | 1.41 | 🤍       |
+
+### Security Level Legend
+| Heart | Description                                               |
+| ----- | --------------------------------------------------------- |
+| 💚    | No successful attacks                                     |
+| 🤍    | Obsoleted by another algorithm                            |
+| 💛    | Theoretical break: security lower than claimed            |
+| 💔    | Attack demonstrated in practice: avoid if at all possible |
```
Comment (tarcieri) [inline README.md:37]: Not sure if these are useful if they aren't in the table

Comment (somehybrid) [reply README.md:37]: They might be reserved for a later date if any of them are attacked.

File: README.md [hunk 2]
```diff
@@ -16,17 +16,25 @@ crate.
...
+| [`chacha20poly1305`] | [(X)ChaCha20Poly1305] | [![crates.io](https://img.shields.io/crates/v/chacha20poly1305.svg)](https://crates.io/crates/chacha20poly1305) | [![Documentation](https://docs.rs/chacha20poly1305/badge.svg)](https://docs.rs/chacha20poly1305) | 1.56 | 💚       |
+| [`deoxys`]           | [Deoxys-I/II]         | [![crates.io](https://img.shields.io/crates/v/deoxys.svg)](https://crates.io/crates/deoxys)                     | [![Documentation](https://docs.rs/deoxys/badge.svg)](https://docs.rs/deoxys)                     | 1.51 | 💚       |
+| [`eax`]              | [EAX]                 | [![crates.io](https://img.shields.io/crates/v/eax.svg)](https://crates.io/crates/eax)                           | [![Documentation](https://docs.rs/eax/badge.svg)](https://docs.rs/eax)                           | 1.41 | 💚       |
+| [`mgm`]              | [MGM]                 | [![crates.io](https://img.shields.io/crates/v/mgm.svg)](https://crates.io/crates/mgm)                           | [![Documentation](https://docs.rs/mgm/badge.svg)](https://docs.rs/mgm)                           | 1.41 | 🤍       |
```
Comment (newpavlov) [inline README.md:29]: Why it's marked by 🤍? AFAIK it's the only AEAD algorithm standardized in Russia.

**Comment by newpavlov** [comment]
Overall, I am not sure about usefulness of this column in the current form. Right now it only indicates that CCM is an obsolete algorithm. Meanwhile, I personally would "recommend" to use SIV instead of GCM.

**Comment by newpavlov** [comment]
Closing this for now. We need to develop a proper policy for recommending AEAD algorithms first.

**Pull request closed without merging**